.github/workflows/testing.yml

```
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.8, 3.9]
```

```suggestion
        python-version: [3.8, 3.9, 3.10, 3.11]
```

Extending testing versions